### ISSUE TITLE
Dataset availability

### ISSUE THREAD

**Issue by nmstoker** [microsoft/OmniParser#32]
Thank you for this impressive work -really interesting. 

I had a look at the paper, blog post and here, but I cannot see any indication of where the dataset is published - is it available? 

I was interested in checking the dataset because I'm seeing several cases where "OK" buttons get classified as icons wth "a button to close or cancel an action" which seems semantically incorrect: OK is more about proceeding, yet this suggests not proceeding.

It would be great to explore examples in the dataset and potentially add more to try to reduce this outcome, but obviously one needs access before that's feasible 🙂

If it's not available now but you plan to put it out, would be great if you could share a rough timeframe (eg just a few days, weeks or longer).

Many thanks!

**Comment by abrichr** [comment]
@nmstoker the dataset is automatically downloaded in [`download.py`](https://github.com/microsoft/OmniParser/pull/52/files#diff-0754e2389b0a4982ce0106c260cf218699e6f2061105043cc24acc7398d1d38b) in https://github.com/microsoft/OmniParser/pull/52. This downloads the weights from https://huggingface.co/microsoft/OmniParser.

**Comment by aliencaocao** [comment]
This isnt the dataset but weights.

**Comment by nmstoker** [comment]
@yadong-lu - do you have any details regarding dataset availability?

**Comment by nmstoker** [comment]
Hi @yadong-lu - I saw you commented a few days back on this matter [here](https://github.com/microsoft/OmniParser/issues/78#issuecomment-2472303717)

It's great that options are being explored and I appreciate this likely needs time to work through internal processes.

Do you have a rough idea how long it might reasonably take? Eg a few more weeks or is it more like two or three months? 

Would be good to keep up the momentum whilst there's plenty of attention on this exciting research, but I totally get how large companies can be 🙂

**Comment by Meshwa428** [comment]
Yeah large companies often delay the release of some things if it contains proprietary data.

They might be cleaning it up for the release, the wait is good but the delay is bad 😞

**Comment by Carol-gutianle** [comment]
Still waiting on 2025/2/22

**Comment by rogerslh** [comment]
Still waiting on 2025/3/17

**Comment by Meshwa428** [comment]
Not worth the wait. Just run omni parser 2 on your GPU or get some cloud options. Get a dataset which contains GUI images and just run it through the model and you can have a dataset with omni parser format which is 60%(maybe ~76%) accurate

**Comment by FrancescoSaverioZuppichini** [comment]
still waiting

**Comment by SkalskiP** [comment]
still waiting
